### PR TITLE
EHN: Using native `json` replace `demjson`

### PR DESCRIPTION
**Pull request by mrmetrix4**
POC that flow can work with the native json lib.
BTW: Did some research, we can add `separators=(",", ":")` to dumps if we want it to mimic demjson behavior.

### CODE REVIEW

**Review by Zeroto521 — CHANGES_REQUESTED** [review]
`sys.argv` is a port for Flow Launcher. So the inputting is a standard JSON.

File: flowlauncher/FlowLauncher.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __init__(self):
         self.rpc_request = {'method': 'query', 'parameters': ['']}
         self.debugMessage = ""
         if len(sys.argv) > 1:  # from input to get jsonrpc
```
[the remark this answers is not in the document]
Comment (mrmetrix4) [reply flowlauncher/FlowLauncher.py:20]: debug is currently not working TBH.

File: flowlauncher/FlowLauncher.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import inspect
 import sys
```
[the remark this answers is not in the document]
Comment (mrmetrix4) [reply flowlauncher/FlowLauncher.py]: Fixed.

File: requirements.txt [hunk 3]
```diff
@@ -1 +1 @@
-demjson
```
[the remark this answers is not in the document]
Comment (mrmetrix4) [reply requirements.txt:1]: Fixed.

File: flowlauncher/FlowLauncher.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrmetrix4) [reply flowlauncher/FlowLauncher.py]: All other things are fixed.

**Review by Zeroto521 — APPROVED** [review]
Thanks for committing to this feature.

If there is no more new problem, I will merge this into the master branch.

**Pull request merged**